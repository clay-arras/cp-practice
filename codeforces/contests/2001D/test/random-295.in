1
12
23 9 2 21 5 22 2 8 13 2 6 11
