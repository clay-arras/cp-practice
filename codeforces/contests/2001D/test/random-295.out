10
23 9 21 5 22 2 8 13 6 11
